components/webFeatures.ts

```diff
@@ -115,24 +125,51 @@ webFeaturesRouter.get(
         if (!validateUserAndGv(req, res)) {
             return
         }
-
-        if (!req.query.level && !isNaN(parseInt(req.query.level))) {
-            res.json({ error: "bad level" })
+        if (!req.query.level) {
+            formErrorMessage(
+                res,
+                "The request must contain the level to set the escalation to.",
+            )
+            return
+        }
+        if (
+            isNaN(parseInt(req.query.level)) ||
+            parseInt(req.query.level) <= 0
+        ) {
+            formErrorMessage(res, "The level must be a positive integer.")
             return
         }
 
-        if (!req.query.id && !uuidRegex.test(req.query.id)) {
-            res.json({ error: "bad id" })
+        console.log("Setting level to " + req.query.level)
```

Comment:
It's better to use our logging solution because it's all nicely formatted.

Reply:
Changed to formatted logging.